BrainPortal/app/helpers/scope_helper.rb

```diff
@@ -681,13 +681,10 @@ def self.model_filter_values(model, attribute, label, association: nil)
     # Fetch the main filter values as an array of arrays:
     # [[value, label, count], [...]]
     model
-      .where("#{attribute} IS NOT NULL")
```

Comment:
It does leave the possibility of `NULL`s messing up the counts, however.
It was included in the original filtering system, and included here as a safety measure.

I would guess the production environment doesn't have `NULL`s in filterable columns anyway.